classes/patterns/class-issues.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Issues class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Issues pattern includes:
+ * Column, Image, Heading, Paragraph, Media & Text.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Issues extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/issues';
+	}
+
+	/**
+	 * Returns the template for one column.
+	 */
+	public static function get_column_template(): string {
+		return '
+			<!-- wp:column {"verticalAlignment":"center","width":"25%","backgroundColor":"white"} -->
+			<div class="wp-block-column is-vertically-aligned-center has-white-background-color has-background" style="flex-basis:25%">
+				<!-- wp:group {"layout":{"wideSize":"90%","contentSize":"90%"}} -->
+				<div class="wp-block-group">
+					<!-- wp:spacer {"height":"16px"} -->
+					<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer -->
+					<!-- wp:media-text {"mediaWidth":20,"mediaSizeSlug":"thumbnail","isStackedOnMobile":false,"imageFill":false} -->
+					<div class="wp-block-media-text alignwide" style="grid-template-columns:20% auto">
+					<figure class="wp-block-media-text__media"></figure>
+					<div class="wp-block-media-text__content">
+					<!-- wp:paragraph {"align":"left","placeholder":"' . __( 'Enter text', 'planet4-blocks-backend' ) . '",
+					"style":{"typography":{"fontSize":"1rem","fontStyle":"normal","fontWeight":"700"}}} -->
+					<p class="has-text-align-left" style="font-size:1rem;font-style:normal;font-weight:700"></p>
+					<!-- /wp:paragraph --></div></div>
+					<!-- /wp:media-text -->
+					<!-- wp:spacer {"height":"16px"} -->
+					<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer --></div>
+				<!-- /wp:group --></div>
+			<!-- /wp:column --></div>
+		';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:group {"backgroundColor":"grey-10","className":"block-wide"} -->
```

Comment:
We should now use the core class `alignfull` class instead of `block-wide` (see this [previously merged PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/820)). I think that you should also put the `block` class since [it adds our spacing values](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/_blocks.scss) 🤔